backend/src/schema/resolvers/comments.spec.js

```diff
@@ -267,6 +267,50 @@ describe('ManageComments', () => {
           client.request(updateCommentMutation, updateCommentVariables),
         ).resolves.toEqual(expected)
       })
+
+      it('throw an error if an empty string is sent from the editor as content', async () => {
+        updateCommentVariables = {
+          id: 'c456',
+          content: '<p></p>',
+        }
+
+        await expect(client.request(updateCommentMutation, updateCommentVariables)).rejects.toThrow(
+          'Comment must be at least 1 character long!',
+        )
+      })
+
+      it('throws an error if a comment sent from the editor does not contain a single character', async () => {
```

Comment:
Isn’t this just the same as the one before? @mattwr18

Reply:
hahaha, the issue with copying and pasting! :stuck_out_tongue_closed_eyes: